Gopkg.lock

```diff
@@ -539,6 +539,14 @@
   pruneopts = "UT"
   revision = "90fa682c2a6e6a37b3a1364ce2fe1d5e41af9d6d"
 
+[[projects]]
```

Comment:
It looks like you are editing Gopkg.lock manually? You are not supposed to :) See https://golang.github.io/dep/docs/Gopkg.lock.html

Use `dep ensure` instead.

Reply:
Not at all. I know that this file shouldn't be edited manually and I have never done this (at least not intentionally). 
This section was added as part of `dep ensure` process - it is because I started using a new project for mocking HTTP calls in tests. It is [gock](https://github.com/h2non/gock) and from my point of view, it is really powerful and easy to use. My idea is to migrate all tests (in the tenant project) that use `go-vcr` to `gock` in the future. For now, I just started using it as part of this PR.

Reply:
@MatousJobanek I see no dependency added in `Gopkg.toml` file for `gopkg.in/h2non/gock.v1`.  I guess, we required to add it, right?

Reply:
I'm not sure if we required, but I definitely wanted to add it there, but I either forgot or it got lost after the several rebase/merge operations. 
Anyway, it's been added here: 38cf717
Thanks for pointing it out

Reply:
BTW gock looks interesting. Might be a better alternative to go-vcr indeed.